### PR TITLE
Sketching out 1.0 docs refresh.

### PR DESCRIPTION
**Pull request by tomchristie**
**...**

*Hmmm... I've approached this in a non-review friendly way, sketching out the re-write as I go as a thinking process. Might need to take a different tack, to be discussed. Still. Throwing it up for some public visibility.*

### CODE REVIEW

**Comment by T-256** [comment]
Can you publish preview docs?

Also, I think we can automate docs preview publish on PRs.
We could create new workflow for docs and exclude it from _test suite_ workflow (no duplicated docs builds). the new workflow also would automatically preview-publish to gh pages if there were any modifications to docs.

File: docs/community/code_of_conduct.md
Comment (T-256) [inline docs/community/code_of_conduct.md:1]: I'd recommend move to repo's main directory.

File: docs/community/contributing.md
Comment (T-256) [inline docs/community/contributing.md:1]: same

**Comment by tomchristie** [comment]
> Can you publish preview docs?

I'm not sure, how would I do that?

**Pull request closed without merging**